.pre-commit-config.yaml

```diff
@@ -1,6 +1,20 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
 repos:
--   repo: https://github.com/mwouts/jupytext
-    rev: v1.14.1  # CURRENT_TAG/COMMIT_HASH
-    hooks:
-    - id: jupytext
-      args: [--sync]
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.2.0
+  hooks:
+  - id: check-yaml
+  - id: check-added-large-files
```

Comment:
I actually had them in originally, I removed them because of the huge amount of changes all over the place that they would require. should I commit those changes as part of this PR as well?

Reply:
I see, does this affect the `workshops/` folder again or also `analyses/`? The former acts as more of a long term storage for material shown previously and we don't necessarily want to change it a lot, but for the latter I'd be happy to just apply all the changes.

Reply:
Given that this might cause a lot of conflicts though it might be best to wait with that, it might make rebasing #158 a lot more difficult.